docs/UserGuide.md

```diff
@@ -1,20 +1,28 @@
 ---
 layout: page
-title: ProductiveNUS
+title: User Guide
 ---
-
-ProductiveNUS is a **desktop application for managing and scheduling your academic tasks, optimized for use via a Command Line Interface** (CLI) contained in a **Graphical User Interface** (GUI). ProductiveNUS allows you to keep track of all your lessons and assignments at hand and type in your assignments quickly so that no time is wasted.
+## Table of Contents
+{toc:}
 
 --------------------------------------------------------------------------------------------------------------------
+## Introduction
+Welcome to ProductiveNUS User Guide! :blue_book:
 
-## Table of Contents
-1. Getting started
-2. Features
-3. FAQ
-4. Summary of commands supported
+**ProductiveNUS is a desktop application** made for you, **a School of Computing (SoC) student in National University of Singapore (NUS)**, to **manage and schedule your academic tasks more effectively**. It makes use of a **Graphical User Interface (GUI)**, which provides you with an intuitive interface and immediate visual feedback. ProductiveNUS uses a **Command Line Interface (CLI)**; this means that you operate the application by typing commands into a [Command line](#gui-terminologies). If you are fast at typing, you can manage your academic tasks more efficiently.
+
+As a **student from the SoC in NUS**, you tend to have a **heavy workload**. ProductiveNUS helps **improve your productivity** by **enhancing your organisational skills**. Apart from simply **keeping track of your tasks**, ProductiveNUS is capable of **scheduling** them for you so you will never **miss any deadlines**. ProductiveNUS is also compatible with NUSMods, meaning all your **timetable information can be imported easily** into the application so all your academic tasks can be found in just one application.
 
 --------------------------------------------------------------------------------------------------------------------
+## About
+This user guide provides you with the necessary information on how to become an expert user of ProductiveNUS. 
+Before moving on to the next section, [Getting started](#getting-started), you can familiarize yourself with the terminologies, syntax and icons used in this user guide by reading the following sub-sections.  
```

Comment:
Do you think it's better to remove the "Before moving on to the ..." and go straight to "You can familiarise yourself ..."? The first part seems a little redundant to me. Unless you want to prevent users from "skipping ahead" of the user guide :)

Reply:
Also, remember to change to "familiarise"